src/features/hotspots/settings/discovery/dedupeDiscoveryResponses.ts

```diff
@@ -0,0 +1,9 @@
+import { DiscoveryResponse } from '../../../../store/discovery/discoveryTypes'
+
+export default (responses: DiscoveryResponse[]) =>
+  responses.filter(
```

Comment:
This is the bulk of the change. We used to filter out "self" here.

Reply:
would [lodash/uniqBy](https://lodash.com/docs/4.17.15#uniqBy) be useful here? sometimes they use clever algorithms under the hook for these utilities -- was just thinking that filter + findIndex might create a few loops through the set.

Reply:
Yeah, I think so. If nothing else, it makes the code more readable.